openforcefield/tests/test_molecule.py

```diff
@@ -372,6 +372,59 @@ def test_to_networkx(self, molecule):
         """Test conversion to NetworkX graph."""
         graph = molecule.to_networkx()
 
+    @pytest.mark.parametrize('molecule', mini_drug_bank())
+    def test_to_inchi(self, molecule):
+        """Test conversion to standard and non-standard InChI"""
+        inchi = molecule.to_inchi()
+        non_standard = molecule.to_inchi(True)
+
+    @pytest.mark.parametrize('molecule', mini_drug_bank())
+    def test_to_inchikey(self, molecule):
+        """Test the conversion to standard and non-standard InChIKey"""
+        inchikey = molecule.to_inchikey()
+        non_standard_key = molecule.to_inchikey(True)
+
+    def test_from_bad_inchi(self):
+        """Test building a molecule from a bad InChI string"""
+
+        inchi = 'InChI=1S/ksbfksfksfksbfks'
+        with pytest.raises(RuntimeError):
+            mol = Molecule.from_inchi(inchi)
+
+    inchi_data = [{'molecule': create_ethanol(), 'standard_inchi': 'InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3',
+                   'fixed_hydrogen_inchi': 'InChI=1/C2H6O/c1-2-3/h3H,2H2,1H3'},
+                  {'molecule': create_reversed_ethanol(), 'standard_inchi': 'InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3',
+                   'fixed_hydrogen_inchi': 'InChI=1/C2H6O/c1-2-3/h3H,2H2,1H3'},
+                  {'molecule': create_acetaldehyde(), 'standard_inchi': 'InChI=1S/C2H4O/c1-2-3/h2H,1H3',
+                   'fixed_hydrogen_inchi': 'InChI=1/C2H4O/c1-2-3/h2H,1H3'},
+                  {'molecule': create_cyclohexane(), 'standard_inchi': 'InChI=1S/C6H12/c1-2-4-6-5-3-1/h1-6H2',
+                   'fixed_hydrogen_inchi': 'InChI=1/C6H12/c1-2-4-6-5-3-1/h1-6H2'}
+                  ]
+
+    @pytest.mark.parametrize('data', inchi_data)
+    def test_from_inchi(self, data):
```

Comment:
Could you move this test to `test_toolkits` and parametrize it to run separately on RDKit and Openeye?